pkg/middleware/request_metrics.go

```diff
@@ -50,11 +50,6 @@ func RequestMetrics(features featuremgmt.FeatureToggles) web.Handler {
 	log := log.New("middleware.request-metrics")
 
 	return func(res http.ResponseWriter, req *http.Request, c *web.Context) {
-		if strings.HasPrefix(c.Req.URL.Path, "/public/") || c.Req.URL.Path == "robots.txt" || c.Req.URL.Path == "/metrics" {
```

Comment:
I removed this guard since the overall compute of instrumenting /robots.txt and /metrics requests is very low and all requests for public assets goes under the same handler.

Reply:
not sure if it's a problem but public requests are now instrumented as unknown 
```
logger=middleware.request-metrics t=2022-06-10T17:26:21.796337+03:00 level=warn msg="request instrumented as unknown" path=/public/fonts/roboto/RxZJdnzeo3R5zSexge8UUVtXRa8TVwTICgirnJhmVJw.woff2 status_code=200
logger=middleware.request-metrics t=2022-06-10T17:26:21.797021+03:00 level=warn msg="request instrumented as unknown" path=/public/build/runtime~app.2bbbf316819314dfeebe.js status_code=200
logger=middleware.request-metrics t=2022-06-10T17:26:21.79856+03:00 level=warn msg="request instrumented as unknown" path=/public/build/grafana.dark.9d7c2bf645cce6645e0a.css status_code=200
logger=middleware.request-metrics t=2022-06-10T17:26:21.912355+03:00 level=warn msg="request instrumented as unknown" path=/public/img/grafana_icon.svg status_code=200
logger=middleware.request-metrics t=2022-06-10T17:26:21.912365+03:00 level=warn msg="request instrumented as unknown" path=/public/fonts/roboto/CWB0XYA8bzo0kSThX0UTuA.woff2 status_code=200
logger=middleware.request-metrics t=2022-06-10T17:26:22.418952+03:00 level=warn msg="request instrumented as unknown" path=/public/build/app.af2f5b8739b3635426e7.js status_code=200
logger=middleware.request-metrics t=2022-06-10T17:26:23.06259+03:00 level=warn msg="request instrumented as unknown" path=/public/img/apple-touch-icon.png status_code=200
logger=middleware.request-metrics t=2022-06-10T17:26:23.062768+03:00 level=warn msg="request instrumented as unknown" path=/public/img/fav32.png status_code=200
logger=middleware.request-metrics t=2022-06-10T17:26:24.172719+03:00 level=warn msg="request instrumented as unknown" path=/public/build/AngularApp.8b385045b7ff22e77147.js status_code=200
logger=middleware.request-metrics t=2022-06-10T17:26:31.631842+03:00 level=warn msg="request instrumented as unknown" path=/favicon.ico status_code=302
```

Reply:
Yeah was noticing that as well. No regexp for public

Reply:
thanks! Fixed in latest commit.